db/sqldb/migrations/0013_user_search.up.sql

```diff
@@ -0,0 +1,9 @@
+BEGIN;
+
+CREATE EXTENSION IF NOT EXISTS pg_trgm;
```

Comment:
This will explode on Azure if not configured there first, I've [gone ahead and set it in the portal](https://learn.microsoft.com/en-us/azure/postgresql/flexible-server/concepts-extensions), and I'll send you a Terraform PR as well.

I'm not sure if it's required locally. If not, you can remove this entirely

Reply:
OK, sounds good. Looks like it still works without it!